language-sql.md

```diff
@@ -3,11 +3,33 @@
 ### Minimum Requirements
 
 - MySQL keywords in uppercase
-- Column names in lowercase
+    E.g.: 
+    ```
+    UPDATE `consumer` SET `PASSWORD` = MD5(`PASSWORD`);
+    ```
+
+- Database table names must be plural. 
+    E.g.: 
+    `users`, `subscriptions`
+    
+    for Linking tables :
+    `users_subscriptions`
+
+- Column names in lowercase and must be singular. Avoid using MySQL reserved words such as order, status, etc.
+    E.g.: `username`, `is_active`, `created_by`, `created_at`
+    
+    - Foreign keys must be prefix with parent table.
+    E.g.:
+    `country_id`
+    
+- Table and column names must be enclosed with ` (backtick operator)
```

Comment:
Table and column names **MUST** be enclosed with <code>`</code> (backtick operator).

You can use `<code>…</code>` tag in markdown if you want to show the code for `.